src/org/labkey/test/components/ui/navigation/apps/ProductListItem.java

```diff
@@ -44,6 +44,11 @@ public void clickNavIcon()
         elementCache().navIcon.click();
     }
 
+    public boolean isEnabled()
+    {
+        return !"labkey-page-nav-disabled".equals(_el.getAttribute("class"));
```

Comment:
Do we have to worry about the class attribute having anything more than "labkey-page-nav-disabled"? If so might change this to:
`return !_el.getAttribute("class").contains("labkey-page-nav-disabled");`

Reply:
Currently, equals works, but checking contains is probably better for future-proofing.